### DISCUSSION TITLE
Different results between New Lagrangian Kernels and old system

### DISCUSSION THREAD

**Discussion by rh201** [idaholab/moose#24067] [Q&A Modules: Solid mechanics] [answered]
Hello MOOSEer,

I'm trying to compare the New Lagrangian Kernels with the old ones. I found that for small strain (or large_kinematics = false), the results of uniaxial tension are the same, but for finite strain (or large_kinematics = true), the results are different, as shown in the figure below. Are the results reasonable or do I misunderstand somewhere? Here are the input files I used. Thanks. https://github.com/rh201/New-Lagrangian-Kernels.git
![f](https://user-images.githubusercontent.com/79764031/232011732-febcf3eb-afa4-451e-8771-3d09bdd60261.png)

**Comment by hugary1995** [comment, marked as answer]
There are three objective rates that come with the new system. The default is Truesdell IIRC. Can you also compare Jaumann and Green-Naghdi rates on the same plot?

Short answer is all models are wrong, some are useful. Pick the one most suitable for your problem. Otherwise, pick your favorite.

**Reply by rh201** [reply]
Thanks @hugary1995 for the advice. Here I plot the results with three different objective_rate, and the other two ways works perfect.
![f0](https://user-images.githubusercontent.com/79764031/232053861-6507deac-24dd-4107-bca9-39965a365a81.png)
